modules/docs/lib/widgets/enhancedComponent.tsx

```diff
@@ -122,7 +129,9 @@ registerWidget<EnhancedComponentValue>('enhancedComponent', ({value, doc, meta})
         ? value.subComponents.map((c, i) => {
             return (
               <React.Fragment key={i}>
-                <Heading>
+                <Heading
+                  id={`${value.displayName?.toLocaleLowerCase()}-${c.name.toLocaleLowerCase}`}
```

Comment:
Something's off with the ids assigned to subcomponents:
<img width="1845" alt="Screenshot 2023-07-12 at 12 55 20 PM" src="https://github.com/Workday/canvas-kit/assets/146020/62bbdd73-4269-4aee-85e8-1a8c67c038d6">

Reply:
yep!